domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -36,22 +36,22 @@ enum class ReviewScheduleCalculationRule {
   PRISONER_UN_SENTENCED,
 }
 
-enum class ReviewScheduleStatus(val inScope: Boolean, val isExclusion: Boolean = false, val isExemption: Boolean = false) {
-  SCHEDULED(true),
-  EXEMPT_PRISONER_DRUG_OR_ALCOHOL_DEPENDENCY(false, isExclusion = true),
-  EXEMPT_PRISONER_OTHER_HEALTH_ISSUES(false, isExclusion = true),
-  EXEMPT_PRISONER_FAILED_TO_ENGAGE(false, isExemption = true),
-  EXEMPT_PRISONER_ESCAPED_OR_ABSCONDED(false, isExemption = true),
-  EXEMPT_PRISONER_SAFETY_ISSUES(false, isExclusion = true),
-  EXEMPT_PRISON_REGIME_CIRCUMSTANCES(false, isExclusion = true),
-  EXEMPT_PRISON_STAFF_REDEPLOYMENT(false, isExemption = true),
-  EXEMPT_PRISON_OPERATION_OR_SECURITY_ISSUE(false, isExemption = true),
-  EXEMPT_SECURITY_ISSUE_RISK_TO_STAFF(false, isExclusion = true),
-  EXEMPT_SYSTEM_TECHNICAL_ISSUE(false), // system down
-  EXEMPT_PRISONER_TRANSFER(false, isExemption = true),
-  EXEMPT_PRISONER_RELEASE(false, isExemption = true),
-  EXEMPT_PRISONER_DEATH(false, isExemption = true),
-  COMPLETED(false),
+enum class ReviewScheduleStatus(val isExclusion: Boolean = false, val isExemption: Boolean = false) {
```

Comment:
As with `InductionScheduleStatus` above I've removed the `inScope` property as we dont use it and most likely wont. It was added initially because we thought being in or out of scope was something we were interested in, but we now know the concept of being in or out of scope is more relevant to Curious.